### DISCUSSION TITLE
Audacity macOS PKG Installer

### DISCUSSION THREAD

**Discussion by LuckySlam** [audacity/audacity#4103] [Ideas & Feature Requests]
Hi,

Can you provide us a universal **PKG** installer ?

We would like to deploy Audacity trought our Microsoft Intune Company Portal for Mac users.

We actualy can deploy it but we first need to pkgbuild the app and sign the PKG with our own Apple Dev ID...and repeat for each versions.

<img width="362" alt="image" src="https://user-images.githubusercontent.com/78394176/209173724-ed2e12bc-94ef-4522-85ec-0890047efe39.png">

Thank you :)

Luc

**Comment by LWinterberg** [comment]
There were problems with universal builds in the past which is why the universal builds weren't included in the release even though existed. Allegedly these issues are fixed now, so 3.2.3 should have a universal option

**Reply by crsib** [reply]
Still, the problem in question is about having PKG installer along with DMG

**Comment by LuckySlam** [comment]
Still no news...
So...we will keep packaging our own PKG.
